cognite/client/_version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "2.29.0"
-__api_subversion__ = "V20210423"
+__version__ = "2.29.1"
+__api_subversion__ = "V20210818"
```

Comment:
```suggestion
__version__ = "2.30.0"
__api_subversion__ = "V20210423"
```

This looks like a rather big change than patch. Also are we following alpha/beta versioning on this? if not we can leave subversion be. :)

Reply:
Right, I've become biased by the NPM way of doing things, where patch is used for any non breaking things, while minor is used for minor breaking changes, and major for big breaking changes. I'll fix it.